### PR TITLE
Address issue #103

### PR DESCRIPTION
**Pull request by marsbroshok**
Address issue #103 
In 'merge.py' we need to check shapely object's class before doing the orientation correction:
* if it's MultiPolygon, then we need to iterate over a MultiPolygon's polygons and manually orient them
* otherwise just orient the Polygon

### CODE REVIEW

File: robosat/tools/merge.py [hunk 1]
```diff
@@ -66,8 +66,15 @@ def unbuffered(shape):
 
         if merged.is_valid:
             # Orient exterior ring of the polygon in counter-clockwise direction.
-            merged = shapely.geometry.polygon.orient(merged, sign=1.0)
-
+            if (isinstance(merged, shapely.geometry.polygon.Polygon)):
```
[the remark this answers is not in the document]
Comment (marsbroshok) [reply robosat/tools/merge.py]: Thank you, I removed redundant parentheses

**Comment by daniel-j-h** [comment]
Perfect, thank you! I'm merging this in and I will ping the shapely folks to ask if we should get this upstream.

**Comment by daniel-j-h** [comment]
This should indeed go upstream after talking to shapely folks; I opened https://github.com/Toblerity/Shapely/issues/635

**Comment by marsbroshok** [comment]
Perfect! Thanks for your great work on Robosat 👍

**Pull request merged**